assets/resources/infrared/assets/projectors.ir

```diff
@@ -0,0 +1,778 @@
+Filetype: IR library file
+Version: 1
+#
+# ON
+name: POWER
```

Comment:
Button naming doesn't match our standard schema

Reply:
Fixed, and name renamed to lowercase to fit other .irs